### PR TITLE
[EASI-2217] Read-only Model Basics - FE

### PR DESCRIPTION
**Pull request by garyjzhao**
# EASI-2217

## Changes and Description

- Creates new Model Basics Read Only component 
- Creates new reusable  `ReadOnlySection` component to be used throughout Read Only pages
- Creates new query and type for this page `src/queries/ReadOnly/GetAllBasics.ts`
- Adds `translateModelType` to the giant `utils/modelPlan.ts` aka translation file
- Adds a few new copy updates to `i18n` files
- Separated `TaskListStatusTag` inside of `TaskListItem` file so I can use the tag on this page too

## Acceptance Criteria
- Render questions and answers from the task list section
- Should still render unanswered questions - populated with 'No answer entered'
- Testing

## How to test this change

1. Navigate to the Model Summary ( Models on nav > Complete Plan > View this Model Plan on sidebar)
2. Ensure data is displayed correctly and matches figma

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Tested user-facing changes with voice-over and the [rotor menu](https://support.apple.com/guide/voiceover/with-the-voiceover-rotor-mchlp2719/mac)

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: src/views/ModelPlan/ReadOnly/ModelBasics/index.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  border-left-width: 0.3rem !important;
+
+  &::before {
+    content: '' !important;
+    width: 1.1rem !important;
+    height: 1.1rem !important;
+    left: 10px !important;
+    margin-top: 0 !important;
+  }
+}
```
Comment (garyjzhao) [inline src/views/ModelPlan/ReadOnly/ModelBasics/index.scss:11]: Had to forcefully remove content (the numbers) and adjust its size and alignment to change  the Process List bullets to match figma

|Stock|Modified|
|---|---|
|image|image|

File: src/views/ModelPlan/ReadOnly/ModelBasics/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          <Route path="/models/:modelID/read-only/model-basics">
+            <ReadOnlyModelBasics modelID={modelID} />
+          </Route>
+        </MockedProvider>
+      </MemoryRouter>
+    );
+    await waitFor(() => {
+      expect(asFragment()).toMatchSnapshot();
+    });
+  });
```
Comment (patrickseguraoddball) [inline src/views/ModelPlan/ReadOnly/ModelBasics/index.test.tsx:107]: This snapshot isn't capturing any of the query data.  It's generally a good idea to inspect the snapshot to insure the query data is baked in.  For all snapshot that render async data, you need to await the data before asserting the snapshot.  You need to make sure you await the queried data, and not some static component.

Something like this:

```
await waitFor(() => {
      expect(
        screen.getByTestId('model-category')
      ).toHaveValue('Primary Care');
});

expect(asFragment()).toMatchSnapshot();
```

File: src/views/ModelPlan/ReadOnly/__snapshots__/index.test.tsx.snap [hunk 3]
```diff
@@ -457,9 +457,44 @@ exports[`Read Only Model Plan Summary matches snapshot 1`] = `
...
+                    >
+                      <h1
+                        aria-live="polite"
+                        class="mint-h1"
+                        tabindex="-1"
+                      >
+                        This page cannot be found.
+                      </h1>
+                      <p>
+                        Here is a list of things you could try to check and fix:
```
Comment (patrickseguraoddball) [inline src/views/ModelPlan/ReadOnly/__snapshots__/index.test.tsx.snap:471]: This snapshot should contain valid data

**Review by patrickseguraoddball — CHANGES_REQUESTED** [review]
Nice work Gary!  I like the component for readonly.  Just some issues with the snapshots and baking in the query data.

**Review by patrickseguraoddball — DISMISSED** [review]

**Review by patrickseguraoddball — APPROVED** [review]

**Pull request merged**